src/node/internal/events.ts

```diff
@@ -520,8 +520,7 @@ function _addListener(
         }
       );
       // Only the newer process version compat adds process.emitWarning support.
-      if (Cloudflare.compatibilityFlags['enable_nodejs_process_v2'])
```

Comment:
Revert this to use `Cloudflare.compatibilityFlags`

Reply:
Why? I've already added a TODO and stated that I'll address this in a follow up?

Reply:
I'll address this in a follow up. I've added a TODO to the compatibility_flags.d.ts file